samples/wifi/raw_tx_packet/Kconfig

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Raw Tx Packet sample"
+
+config CONNECTION_TIMEOUT
+	int "Time to wait for a station to connect"
+	default 30
+
+config RAW_TX_PACKET_SAMPLE_SSID
+	string "SSID"
+	help
+	  Specify the SSID to connect
```

Comment:
@D-Triveni  is this really the SSID to connect to?

Reply:
If so, maybe add some more help text, i.e. that it is needed for the Connected Station mode, etc.

Reply:
Yes @ioannisg , we use this SSID to connect to an AP. Will add more details in help text.